src/mlj_flux.jl

```diff
@@ -328,8 +414,6 @@ function MLJFlux.fit!(model::LaplaceRegression, verbosity, X, y)
         # train the model
         for (X_batch, y_batch) in loader
             y_batch = reshape(y_batch, 1, :)
-            println("Shape of X_batch dopo di loader: ", size(X_batch))
-            println("Shape of y_batch dopo di loader : ", size(y_batch))
 
             # Backward pass
             gs = Flux.gradient(parameters) do
```

Comment:
Is there a reason for no longer relying on `MLJFlux.train!` like [here](https://github.com/JuliaTrustworthyAI/LaplaceRedux.jl/blob/d7d0ad7f6faaa1dd729e194150171413eadc4a88/src/mlj_flux.jl#L193)?

Reply:
So it doesn't break or use a deprecated function once train change

Reply:
`MLJFlux.fit!` is deprecated as of the recent `v0.5.0`, but it can still be used. So either we go straight to a version where we use the new API (`fit!` has become `train` and `train!` has become `train_epochs` or we ignore the deprecations for now. In any case, we should use high-level functions where possible and only overload and go low-level where necessary.